docs/docs/troubleshooting.md

```diff
@@ -224,6 +225,33 @@ ls -la /sys/kernel/btf/vmlinux
 sudo tracee --events execve
 ```
 
+## Security Considerations
```

Comment:
Don't feel it belongs here. Basically documenting a TOCTOU? It is not troubleshooting.